iotdb-1.1/src/main/java/cn/edu/tsinghua/iot/benchmark/iotdb110/IoTDBClusterSession.java

```diff
@@ -19,65 +19,111 @@
 
 package cn.edu.tsinghua.iot.benchmark.iotdb110;
 
+import cn.edu.tsinghua.iot.benchmark.tsdb.DBConfig;
+import cn.edu.tsinghua.iot.benchmark.tsdb.TsdbException;
 import org.apache.iotdb.isession.pool.SessionDataSetWrapper;
 import org.apache.iotdb.rpc.IoTDBConnectionException;
 import org.apache.iotdb.rpc.StatementExecutionException;
 import org.apache.iotdb.session.pool.SessionPool;
 import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
-import org.apache.iotdb.tsfile.read.common.Field;
 import org.apache.iotdb.tsfile.read.common.RowRecord;
 import org.apache.iotdb.tsfile.write.record.Tablet;
-
-import cn.edu.tsinghua.iot.benchmark.client.operation.Operation;
-import cn.edu.tsinghua.iot.benchmark.conf.Config;
-import cn.edu.tsinghua.iot.benchmark.conf.ConfigDescriptor;
-import cn.edu.tsinghua.iot.benchmark.entity.Batch;
-import cn.edu.tsinghua.iot.benchmark.entity.DeviceSummary;
-import cn.edu.tsinghua.iot.benchmark.entity.Record;
-import cn.edu.tsinghua.iot.benchmark.entity.Sensor;
-import cn.edu.tsinghua.iot.benchmark.measurement.Status;
-import cn.edu.tsinghua.iot.benchmark.schema.schemaImpl.DeviceSchema;
-import cn.edu.tsinghua.iot.benchmark.tsdb.DBConfig;
-import cn.edu.tsinghua.iot.benchmark.tsdb.TsdbException;
-import cn.edu.tsinghua.iot.benchmark.workload.query.impl.DeviceQuery;
-import cn.edu.tsinghua.iot.benchmark.workload.query.impl.VerificationQuery;
-import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.sql.SQLException;
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
-import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Executors;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
-import java.util.concurrent.atomic.AtomicBoolean;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.stream.Collectors;
 
 public class IoTDBClusterSession extends IoTDBSessionBase {
+  private class BenchmarkSessionPool implements IBenchmarkSession {
+    private final SessionPool sessionPool;
+    public BenchmarkSessionPool(SessionPool sessionPool) {
+      this.sessionPool = sessionPool;
+    }
+    @Override
+    public void open() {
+      
+    }
+    @Override
+    public void open(boolean enableRPCCompression) {
+      
+    }
```

Comment:
Maybe we need to init SessionPool in BenchmarkSessionPool instead of passing sessionpool into BenchmarkSessionPool